RecoDataProducts/src/classes_def.xml

```diff
@@ -245,6 +245,10 @@
  <class name="mu2e::TrkQualCollection"/>
  <class name="art::Wrapper<mu2e::TrkQualCollection>"/>
 
+ <class name="mu2e::MVAResult"/>
+ <class name="mu2e::MVAResultCollection"/>
+ <class name="art::Wrapper<mu2e::MVAResultCollection>"/>
```

Comment:
Do you expect to put a single MVAResult into the event as a data product, or only collections?  If you want the former, you also need the Wrapper<mu2e::MVAResult>.

Reply:
Done in 2abe4fb

Reply:
Thanks!  One last CI run and I will merge.